app/src/main/java/com/kickstarter/ui/adapters/DiscoveryPagerAdapter.java

```diff
@@ -40,8 +41,12 @@ public void setPrimaryItem(final @NonNull ViewGroup container, final int positio
   }
 
   @Override
-  public @NonNull Fragment getItem(final int position) {
-    return DiscoveryFragment.newInstance(position);
+  public
+  @NonNull
+  Fragment getItem(final int position) {
```

Comment:
looks like something android studio does automatically when auto-completing override methods, cory and i are gonna pair this morning and i'll explain our code style!